app/controllers/application_tokens_controller.rb

```diff
@@ -0,0 +1,43 @@
+# ApplicationTokensController manages the creation/removal of application tokens
+class ApplicationTokensController < ApplicationController
+  respond_to :js
+
+  ## GET /application_tokens/new
+  # def new
+  #  @application_token = ApplicationToken.new
+  # end
+
+  # POST /application_tokens
+  def create
+    @plain_token = Devise.friendly_token
+
+    @application_token = ApplicationToken.new(create_params)
+    @application_token.user = current_user
+    @application_token.token_salt = BCrypt::Engine.generate_salt
+    @application_token.token_hash = BCrypt::Engine.hash_secret(
+      @plain_token,
+      @application_token.token_salt)
+
+    if @application_token.save
+      respond_with @application_token
+    else
+      respond_with @application_token.errors, status: :unprocessable_entity
+    end
+  end
+
+  # DELETE /application_token/1
+  def destroy
+    @application_token = ApplicationToken.find(params[:id])
+    # @team_user.create_activity!(:remove_member, current_user)
```

Comment:
Good catch, I forgot about it. I'm going to track activities with a another commit inside of this PR